requirements/quality.in

```diff
@@ -6,4 +6,5 @@ edx-lint                  # edX pylint rules and plugins
 isort                     # to standardize order of imports
 pycodestyle               # PEP 8 compliance validation
 pydocstyle<2.0            # PEP 257 compliance validation: 2.0 doesn't know Google style, and tries to correct my grammar... :(
-readme_renderer           # Validates README.rst for usage on PyPI
```

Comment:
We don't need this as a direct dependency now after switching to `twine check` instead.